plutus-metatheory/src/Raw.lagda

```diff
@@ -114,6 +122,23 @@ decBuiltin verifySchnorrSecp256k1Signature verifySchnorrSecp256k1Signature = tru
 decBuiltin equalsByteString equalsByteString = true
 decBuiltin appendString appendString = true
 decBuiltin trace trace = true
+decBuiltin bls12-381-G1-add bls12-381-G1-add = true
+decBuiltin bls12-381-G1-neg bls12-381-G1-neg = true
+decBuiltin bls12-381-G1-scalarMul bls12-381-G1-scalarMul = true
+decBuiltin bls12-381-G1-equal bls12-381-G1-equal = true
+decBuiltin bls12-381-G1-hashToGroup bls12-381-G1-hashToGroup = true
+decBuiltin bls12-381-G1-compress bls12-381-G1-compress = true
+decBuiltin bls12-381-G1-uncompress bls12-381-G1-uncompress = true
+decBuiltin bls12-381-G2-add bls12-381-G2-add = true
+decBuiltin bls12-381-G2-neg bls12-381-G2-neg = true
+decBuiltin bls12-381-G2-scalarMul bls12-381-G2-scalarMul = true
+decBuiltin bls12-381-G2-equal bls12-381-G2-equal = true
+decBuiltin bls12-381-G2-hashToGroup bls12-381-G2-hashToGroup = true
+decBuiltin bls12-381-G2-compress bls12-381-G2-compress = true
+decBuiltin bls12-381-G2-uncompress bls12-381-G2-uncompress = true
+decBuiltin bls12-381-millerLoop bls12-381-millerLoop = true
+decBuiltin bls12-381-mulMlResult bls12-381-mulMlResult = true
+decBuiltin bls12-381-finalVerify bls12-381-finalVerify = true
```

Comment:
I'm planning on automating the definition of this function in this sprint.

Reply:
I did wonder if there was some better way to do that. I spent ages rearranging all of the cases so that I could see which ones were missing!